Testing/VelaptorTests/OpenGL/GLWindowTests.cs

```diff
@@ -656,7 +952,7 @@ public void Show_WhenInvoked_SetsUpOpenGLErrorCallback()
             var window = CreateWindow();
             window.Show();
 
-            this.mockWindowFacade.Raise(i => i.Load += null, EventArgs.Empty);
```

Comment:
Verify that the error callback is being setup properly in the unit test `Show_WhenInvoked_SetsUpOpenGLErrorCallback`